src/main/java/org/folio/edge/inventory/controller/InventoryController.java

```diff
@@ -16,6 +18,7 @@
 public class InventoryController implements InventoryApi {
 
   private final InventoryService inventoryService;
+  private final DataExportService dataExportService;
```

Comment:
It's strange to have data-export service in inventory controller.
As well as endpoints
` /inventory/download-authority/{authorityId}` - authority doesn't related to the inventory at all.
And both new endpoints are also not very related to the inventory as the response is MARC records, that is basically an source records.

It's just suggestion to keep everything clean related to what and from where because it could produce unclearness in the future.

Reply:
Hi @psmagin, it was decided to expose such endpoints in edge-inventory to not create a new separate edge module for data-export  as far as it will take more resource to deploy a new additional edge module